README.md

```diff
@@ -1,108 +1,45 @@
 
 # Contents
-#### [Introduction](#introduction)
-1. [Need to Know Terminology](#terminology)
-2. [Why Version Control](#version-control)
-3. [Tutorial](#tutorial)
-	* [Getting Started](#getting-started)
-	* [Branching](#branching)
-	* [Making Changes](#changes)
-	* [Merging with Master](#merging)
-	* [Merge Conflicts](#conflicts)
-	* [Changing File Structure](#changing-file-structure)
-4. [Introducing Github Flow](#github-flow)
-
-#### [Git for Collaboration](#git-collaboration)
-1. [Further terminology](#further-terminology)
-2. [The Timeline](#timeline)
-3. [Committing](#commits)
-    * [When should you commit](#when-commit)
-    * [Commit messages](#commit-message)
-    * [Merging commits](#merging-commits)
-
-#### [Resources](#resources)
-
-
-
-<a id="introduction" name="introduction"></a>
-# Introduction
-
-The introduction to Git is aimed at the first week of the full time Founders & Coders course, or to those just starting out with git. The hope is that this introduction will cover all you need to know to start collaborating on code with your fellow teammates.
-
-
-<a id="terminology" name="terminology"></a>
-## Terminology
-
-##### *Repository (repo)*
-Simply put, this is your project folder. This repository can be located locally (in your file system on your computer), or remotely (on Github). Either way, it is the same repository.
-
-##### *Version Control*
-We will use Git for this. A way of keeping track of changes in the code which makes it possible to work with multiple developers on the same repo.
-
-##### *Git*
-A version control system.
-
-##### *Github*
-A remote location where you can store you code, and which all the members of the team have access to. Think of it like a dropbox for code. One of the big differences however is most repositories on github are public. Anybody can see your code.
-
-##### *Commit*
-A way of saving your code at different points along the project. Unlike many tools you may have used however, all commits are saved. This creates a project history and a way to track changes.
-
-##### *Branches*
-As you work on a git repo the first branch you are on is usually the default branch. This is often called `master`. If you start working on a section of the website (say the footer styling), it is best practise to create your own branch for that feature. Creating your own branch is like taking a copy of `master` and renaming it. When you commit, they will now be on that new branch only.
-
-In the sections below we will walk through how to do this. For the meantime, just note that you always have one default branch, and can have as many other branches as needed.
-
-
-<a name="version-control" id="version-control"></a>
-## Why Version Control
```

Comment:
I find this is the area which people need reminding of the absolute most in the beginning. If possible, could this just be moved to a different markdown file and be said to be optional?